docs/overview/changelog.md

```diff
@@ -7,6 +7,47 @@ up-to-date list of all changes.
 
 ## Zulip Server 9.x series
 
+### Zulip Server 9.3
+
+_Released 2024-11-22_
+
+- The documentation for the Docker image no longer inaccurately
+  describes it as alpha/experimental software.
+- Fixed the migration in 9.2, `0576_backfill_imageattachment.py` to
+  correctly find the files to process for installations using the S3
+  backend, added a progress indicator, and a duplicate migration
```

Comment:
```suggestion
- Fixed the migration in 9.2, `0576_backfill_imageattachment.py`, to
  correctly find the files to process for installations using the S3
  backend; added a progress indicator, and a duplicate migration
```

Reply:
Did a different edit to split sentences, but thanks for observing the weird grammar there.